src/diffusers/pipelines/pipeline_flax_utils.py

```diff
@@ -296,6 +296,7 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
         use_auth_token = kwargs.pop("use_auth_token", None)
         revision = kwargs.pop("revision", None)
         from_pt = kwargs.pop("from_pt", False)
+        use_memory_efficient_attention = kwargs.pop("use_memory_efficient_attention", False)
```

Comment:
I decided to enable this at pipeline load time. It's not straightforward to replicate the recursive logic we created for `xformers` because Flax submodules are not available unless you are in `apply`, so I didn't find a way to make functions to enable/disable the setting on demand. Instead, the configuration is set up when the pipeline is instantiated.

Another alternative would have been to pass `use_memory_efficient_attention` as an additional argument to `generate` so it's applied on a per-inference basis. I thought making it constant for the pipeline made sense for the Flax case.

Any other thoughts about this @patrickvonplaten, @yiyixuxu, @williamberman?